docker-compose.example.sqlite.yml

```diff
@@ -1,20 +1,26 @@
-version: "3.7"
+version: "2.4"
```

Comment:
Any particular reason for 2.4 vs. some other version? Is that the minimum for the features use by the compose file?

Reply:
2.4 is the latest in the 2.x branch, and fully supports all the options, as well as being Portainer (libcompose) compatible. Once Portainer's main branches are all updated this can probably come up to the 3.x branch, but honestly most of the new compose features in 3.x are heavily oriented toward load sharing/cpu/multicontainer options that probably aren't very important for bb users.